src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Microsoft.VisualStudio.Razor.IntegrationTests.csproj

```diff
@@ -39,4 +36,35 @@
   <ItemGroup>
     <EmbeddedResource Include="**\TestFiles\**\*" />
   </ItemGroup>
+
+  <ItemGroup>
+    <!-- Microsoft.VisualStudio.RazorExtension.vsix cannot be installed without elevation due to a bug in the experimental extension installation logic. -->
+    <!--<AssemblyAttribute Include="Xunit.Harness.RequireExtensionAttribute">
+      <_Parameter1>Microsoft.VisualStudio.RazorExtension.vsix</_Parameter1>
+    </AssemblyAttribute>-->
+    <AssemblyAttribute Include="Xunit.Harness.RequireExtensionAttribute">
+      <_Parameter1>Microsoft.VisualStudio.RazorExtension.Dependencies.vsix</_Parameter1>
+    </AssemblyAttribute>
```

Comment:
📝 This is the part to make conditional if there are cases where we don't want to deploy the dependencies.